src/vs/editor/browser/widget/diffEditorWidget.ts

```diff
@@ -1029,8 +1032,8 @@ export class DiffEditorWidget extends Disposable implements editorBrowser.IDiffE
 		let computedRepresentableSize = Math.max(0, computedAvailableSize - 2 * 0);
 		let computedRatio = scrollHeight > 0 ? (computedRepresentableSize / scrollHeight) : 0;
 
-		let computedSliderSize = Math.max(0, Math.floor(layoutInfo.contentHeight * computedRatio));
-		let computedSliderPosition = Math.floor(scrollTop * computedRatio);
+		let computedSliderSize = Math.max(0, Math.round(layoutInfo.contentHeight * computedRatio));
```

Comment:
Why is `Math.round` preferable to `Math.floor` here? Do you have a screenshot showing a case where this improves something?